rdza_proc_macro/src/lib.rs

```diff
@@ -62,13 +62,13 @@ fn replace_ident(ident: Ident) -> Option<TokenTree> {
         "niech" => "let",
         "statyczny" | "statyczna" | "statyczne" => "static",
         "struktura" => "struct",
-        "spodziewaj_się" => "expect",
+        "spodziewaj_się" | "oczekuj" => "expect",
         "dopóki" => "while",
         "użyj" => "use",
         "do" => "into",
         "rób" => "do",
         "prawda" => "true",
-        "enumeracja" => "enum",
+        "enumeracja" | "wyliczanie" | "wyliczanka" | "wyliczaj" => "enum",
```

Comment:
Moim zdaniem `wyliczaj` tutaj nie pasuje, bo to czasownik, a reszta to rzeczowniki.

```suggestion
        "enumeracja" | "wyliczanie" | "wyliczanka" => "enum",
```